### PR TITLE
Layout of Auth Page on smaller phones is wack

### LINKED ISSUE #93: Layout of Auth Page on smaller phones is wack
- [ ] Resize the text to fit screen 
[How to dynamically resize text in Flutter? - Stack Overflow](https://stackoverflow.com/questions/50751226/how-to-dynamically-resize-text-in-flutter) 

Either 

- [ ] put the whole page in a single child scrollview 

OR 

- [x] layout so the whole page is always visible 
[Creating responsive apps - Flutter](https://flutter.dev/docs/development/ui/layout/responsive)

### PR DESCRIPTION
**Pull request by tamari-gray**
Fixes #93

### CODE REVIEW

**Comment by tamari-gray** [comment]
first pr!!!

File: test/widget/auth_widget_test.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import 'package:crowdleague/widgets/auth/auth_page.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:redux/redux.dart';
+
+import '../mocks/image_test_utils.dart';
+
+void main() {
+  group('auth page', () {
```
Comment (nickmeinhold) [inline test/widget/auth_widget_test.dart:12]: could you change auth page to AuthPage?

File: test/widget/auth_widget_test.dart [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+      // Setup the app state with expected values
+      final initialAppState = AppState.init();
+      // Create the test harness.
+      final store = Store<AppState>(appReducer, initialState: initialAppState);
+      final wut = AuthPage();
+      final harness =
+          StoreProvider<AppState>(store: store, child: MaterialApp(home: wut));
+
+      await provideMockedNetworkImages(() async {
```
Comment (nickmeinhold) [inline test/widget/auth_widget_test.dart:24]: this is my bad but could we try without the provideMockedNetworkImages wrapper?  Thinking about it now I reckon it's worth only adding when needed

**Review by nickmeinhold — CHANGES_REQUESTED** [review]
Looks amazing!  Couple of nits :-)

File: test/widget/auth_widget_test.dart (hunk 1 above)
Comment (tamari-gray) [reply test/widget/auth_widget_test.dart:12]: no! na alg :)

File: test/widget/auth_widget_test.dart (hunk 2 above)
Comment (tamari-gray) [reply test/widget/auth_widget_test.dart:24]: shweeeet

**Review by nickmeinhold — APPROVED** [review]
So good - killer first PR, and really important for making a good impression on different phones!

**Comment by nickmeinhold** [comment]
Thanks heaps Tam!!!

**Pull request merged**